### DISCUSSION TITLE
item frame

### DISCUSSION THREAD

**Discussion by EhsanKiller** [GriefPrevention/GriefPrevention#2339] [Q&A]
hi when a place was climed by a person if in claim there is a item frame and there is an item in item frame player can broke the item that in item frame and pick it up its a bug or can if fix it in config of file ?

**Comment by Jikoo** [comment]
Item frames require build trust to modify. If you are seeing item frames not be protected in areas where users do not have sufficient trust and are not ignoring claims, this is most likely another plugin interfering - the [last 3 bug reports for this](https://github.com/GriefPrevention/GriefPrevention/issues?q=is%3Aissue+is%3Aclosed+item+frame) are not reproducible and have gone without any reply from the posters. Please try using only GP. You can use a plugin like EventDebug to help you track down the plugin causing the bad interaction.

**Comment by EhsanKiller** [comment]
ok beacuse for example i added a fence in item frame in my claim a player can click on item frame and give the item in the item frame

**Reply by Jikoo** [reply]
Yes, I understood what you meant. Please follow my instructions.

For clarity:
1) Reproduce with only GP installed, no other plugins.
2) If the problem can be seen with only GP installed, ensure that you are not ignoring claims and in a claim where you are not granted build trust. If the claim is a subclaim, also ensure that its parent does not grant you build trust.
3) If the problem cannot be seen with only GP installed, narrow down the incompatible plugin. EventDebug is a useful tool for this, but you can just remove half of your plugins at a time to check in larger batches.
